platform/security/handler/security-handler-basic/pom.xml

```diff
@@ -88,22 +100,22 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.86</minimum>
+                                            <minimum>0.85</minimum>
                                         </limit>
                                         <limit>
                                             <counter>BRANCH</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.83</minimum>
+                                            <minimum>0.80</minimum>
                                         </limit>
                                         <limit>
                                             <counter>COMPLEXITY</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.72</minimum>
+                                            <minimum>0.66</minimum>
```

Comment:
Coverage numbers shouldn't, in general, go down if below 75%. If we fluctuate between 75-100%, that's acceptable but when below 75% should always try to stay steady or trend upwards.